pkg/agent/route/route_windows.go

```diff
@@ -168,6 +176,89 @@ func (c *Client) DeleteRoutes(podCIDR *net.IPNet) error {
 	return nil
 }
 
+func (c *Client) AddServiceRoutes(svcIP net.IP, gwIP net.IP) error {
+	obj, found := c.hostRoutes.Load(svcIP.String())
```

Comment:
how are agent restarts handled?
For Pod routes, the `Reconcile` method is used and ensure that the `hostRoutes` map is consistent with actual existing routes on the host. But this doesn't take into account these new Service routes.

Reply:
Thank you for reminding. I have updated `Reconcile` to consider services as well.